.cirrus.yml

```diff
@@ -65,3 +66,79 @@ vagrant_task:
     else
       ssh -tt default "sudo -i make -C /vagrant localrootlessintegration"
     fi
+
+task:
+  timeout_in: 30m
+  env:
+    HOME: /root
+    CIRRUS_WORKING_DIR: /home/runc
+    GO_VERSION: "1.16.4"
```

Comment:
Can switch to 1.16.6 while at it.

Reply:
done

Reply:
Looks like not.

Reply:
I was sure I did that change. Second try pushed just no.w